examples-gallery/plot_sliding_block.py

```diff
@@ -0,0 +1,306 @@
+
+# %%
+""""
+sliding a block on a road
+=========================
+A block, modeled as a particle is sliding on a road to cross a
+hill. The block is subject to gravity and speed dependent
+friction. 
+Gravity points in the negative Y direction.
+A force tangential to the road is applied to the block.
+Three objective functions to me minimized may be selected:
+
+- selektion = 0: time to reach the end point is minimized
+- selektion = 1: time to reach the end point is minimized and 
+  the energy consumed is minimized.
+- selektion = 2: energy consumed is minimized.
+
+**Constants**
+
+- m: mass of the block [kg]
+- g: acceleration due to gravity [m/s**2]
+- reibung: coefficient of friction [N/(m*s)]
+ -a, b: paramenters determining the shape of the road.
```

Comment:
```suggestion
- a, b: paramenters determining the shape of the road.
```

formatting error

Reply:
Still not corrected:
![image](https://github.com/csu-hmc/opty/assets/276007/d94142f6-0980-4af1-ace7-ba780c74a3b3)